### PR TITLE
polish_statistic_pages

### PR DESCRIPTION
**Pull request by ppolakiewicz**
I added bootstrap library to leaderboard page, 
add list of last 250 games with input to filter list.
Rename statistican.py to leaderboard.py cause we changed 
functionality of this package.

I know that our leaderboard page still looks very simple 
but it got all of funcionality so mabey I will add some style
later.

### CODE REVIEW

**Review by devdo-eu — APPROVED** [review]
Looks good 2 me

File: playbot/mind.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def __init__(self, log_method):
         """
         self.__log = log_method
         self.__moves = ['NOP()', 'PATCH()', 'SCAN()', 'OVERLOAD()', 'OVERHEAR()', 'EXPLOIT()', 'INFECT()']
-        self.__my_name = f'RandBot'
+        self.__my_name = f'RandBot_{randrange(1,100)}'
```
Comment (devdo-eu) [inline playbot/mind.py]: w8.
I change this line three weeks ago and erased random number part, so it was possible to rank ELO for this kind of bot.
With random name, evaluation of ELO is near to impossible.
My last change was based on the assumption that the same bot cannot participate in a fight against itself - from the point of view of ELO calculation such a game is meaningless.

Another thing: `randrage(1,100)` - why range is so small? Despite drawing a number, probability of the same name is high and will occur for a range of 99 options (on average once in 99 games - so often)

Before my change the draw range was 500 (set by me :P)

**Review by devdo-eu — CHANGES_REQUESTED** [review]

**Comment by ppolakiewicz** [comment]
As we talked offlien, I updated README about prepared bots

**Comment by devdo-eu** [comment]
Please look at `polish_statistic_pages_changed` branch.
I was meaning changes like on `polish_statistic_pages_changed` branch when we talked offline.

**Comment by ppolakiewicz** [comment]
> Please look at `polish_statistic_pages_changed` branch.
> I was meaning changes like on `polish_statistic_pages_changed` branch when we talked offline.

Hmm, I was convinced that moving bots to directories you wanted to do in the next step (PR).
Looks cool, then maybe in order not to waste your hard work, I will close this PR and open a new one from your branch ?

**Comment by ppolakiewicz** [comment]
I cherry-pick your change.
Feel free to approve it 😄

**Review by devdo-eu — APPROVED** [review]

**Pull request merged**